Content.Server/Spreader/KudzuSystem.cs

```diff
@@ -28,13 +29,12 @@ private void OnDamageChanged(EntityUid uid, KudzuComponent component, DamageChan
     {
         // Every time we take any damage, we reduce growth depending on all damage over the growth impact
         //   So the kudzu gets slower growing the more it is hurt.
-        int growthDamage = (int) (args.Damageable.TotalDamage / component.GrowthHealth);
+        var growthDamage = (int) (args.Damageable.TotalDamage / component.GrowthHealth);
         if (growthDamage > 0)
         {
-            GrowingKudzuComponent? growing;
-            if (!TryComp(uid, out growing))
+            if (!HasComp<GrowingKudzuComponent>(uid))
             {
-                growing = AddComp<GrowingKudzuComponent>(uid);
+                AddComp<GrowingKudzuComponent>(uid);
                 component.GrowthLevel = 3;
             }
```

Comment:
EnsureComp has a bool overload which should be used instead.

Reply:
Aha! I thought that bit looked ugly and wanted to use EnsureComp, but didn't see a way to preserve the logic for setting GrowthLevel. Thanks for pointing that out!